docs/pages/access-controls/idps/saml.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Using Teleport as a SAML identity provider
+description: How to configure and use Teleport as a SAML identity provider.
+h1: Teleport as a SAML identity provider
+---
+
+## How to use Teleport as an identity provider
+
+This guide will cover how to use Teleport as an identity provider so that Teleport
+can be used to authenticate to external services. 
+
+## Prerequisites
+
+- Teleport role with access to maintaining `saml_idp_service_provider` resources.  This is available in the default `editor` role.
```

Comment:
Can we be more explicit here? What does `maintaining` mean?

Reply:
I agree "maintaining" is confusing. Can we just get rid of this pre-requisite and make it a part of the guide? Show an example role that has required permissions and say that builtin "editor" role already has one.